docs/source/pages/supported_technologies/index.rst

```diff
@@ -41,6 +41,9 @@ Package Registries
    * - `JFrog Artifactory <https://jfrog.com/artifactory>`_
      - Only projects built with Gradle and publishing to a JFrog Artifactory repo following `Maven layout <https://maven.apache.org/repository/layout.html>`_
      - :doc:`page </pages/supported_technologies/jfrog>`
+   * - `Maven Central Artifactory <https://central.sonatype.com>`_
+     - This registry is searched for artifacts built with Gradle or Maven.
```

Comment:
I think putting it this way might make the reader misunderstand that the Maven Central Artifactory only supports artifacts built with Gradle and Maven.
Perhaps we could say:
```
Only projects built with Gradle or Maven and published to the Maven Central Artifactory.
```

Reply:
See commit [1aabb11](https://github.com/oracle/macaron/pull/471/commits/1aabb111ad2d55425a391d884238aaa9d4f2a8a3)